Pipeliner/Common/ApiError.swift

```diff
@@ -6,9 +6,10 @@
 //
 
 import Foundation
-enum ApiError: Error {
-    case invalidUrl
-    case emptyResponse
-    case invalidDate
-    case configurationNotFound
+enum ApiError: String, Error {
```

Comment:
Please [use](https://stackoverflow.com/a/39176551/2595805) `LocalizedError.errorDescription` for this purpose, instead of `rawValue`